.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/220868604-108e8bbc-4d88-4c3a-8e44-9f634439445f.png)

- [ ] Kindly make your branch name descriptive enough :+1: You can make it like **feature/finalize-budget-app** instead of **feature_branch** to make your pull request professional :blush: